styles/designs/carnival/parts/_chapterintro-shapes.scss

```diff
@@ -87,6 +86,63 @@
     )
 ));
 
+// Chapter Intro parts
+
+// To do: use ChapterIntroSplashFigureShape everywhere. Create shapes for Chapter Intro body 
+// with specific title and for body of margin type. Apply to all themes.
+
+@include create_shape('carnival:::ChapterIntroSplashFigureShape', (
+    _components: (
+        map-merge($ChapterIntro__Container, (
+            _components: (
+                map-merge($ChapterSplash__Figure__Wrapper, (
+                    _components: (
+                        map-merge($ChapterSplash__Figure, (
+                            _components: (
+                                $ChapterSplash__Figure--Bg,
+                                map-merge($ChapterSplash__Media, (
+                                    _components: (
+                                        $ChapterSplash__Img,
+                                    )
+                                )),
+                            )
+                        )),
+                        map-merge($Caption__Container--ChapterIntro, (
+                            _components: (
+                                $ChapterSplash__Caption--Bg,
+                                $Caption__Number,
+                                $Caption__TitleLabel,
+                                $Caption__Title,
+                                $Caption__Text,
+                            )
+                        )),
+                    )
+                )),
+            )
+        )),
+    )
+));
+
+@include create_shape('carnival:::ChapterIntroBodyWithRegularHeaderShape', (
```

Comment:
In Organic Chemistry title "Introduction" is not in line with intro content, it's styled like regular H2 header.